iothub/device/src/IotHubBaseClient.cs

```diff
@@ -116,6 +116,11 @@ public void SetRetryPolicy(IRetryPolicy retryPolicy)
         /// <summary>
         /// Open the client instance. Must be done before any operation can begin.
         /// </summary>
+        /// <remarks>
+        /// This client can be re-opened after it has been closed, but cannot be re-opened after it has
```

Comment:
Super (and perhaps subjective) nit, but `are not persisted` reads a bit awkward to me -- `do not persist` may be better.